paper/paper.md

```diff
@@ -0,0 +1,136 @@
+---
+title: 'rdata: A Python library for R datasets'
+tags:
+  - Python
+  - R
+  - datasets
+  - rda
+  - rds
+authors:
+  - name: Carlos Ramos-Carreño
+    orcid: 0000-0003-2566-7058
+    affiliation: 1
+  - name: Tuomas Rossi
+    orcid: 0000-0002-8713-4559
+    affiliation: 2
+affiliations:
+  - name: Universidad Autónoma de Madrid, Spain
+    index: 1
+  - name: CSC – IT Center for Science Ltd., Finland
+    index: 2
+date: 4 September 2024
+bibliography: paper.bib
+
+---
+
+# Summary
+
+Research work usually requires the analysis and processing of data from different sources.
+Traditionally in statistical computing, the R language has been widely used for this task, and a huge amount of datasets have been compiled in the Rda and Rds formats, native to this programming language.
+As these formats contain internally the representation of R objects, they cannot be directly used from Python, another widely used language for data analysis and processing.
+The library `rdata` allows to load and convert these datasets to Python objects, without the need of exporting them to other intermediate formats which may not keep all the original information.
+This library has minimal dependencies, ensuring that it can be used in contexts where an R installation is not available.
+The capability to write data in Rda and Rds formats is also under development.
+Thus, the library `rdata` facilitates data interchange, enabling the usage of the same datasets in both languages (e.g. for reproducibility, comparisons of results against methods in both languages, or the creation of complex processing pipelines that involve steps in both R and Python).
+
+# Statement of need
+
+The datasets from the CRAN repository are stored in the R specific formats Rda and Rds.
```

Comment:
I re-read the document. I think this sentence should be changed to place less emphasis on CRAN (which should be just an example). Apart from that I do not see anything else that I would like to change, so for my part it would be ready to send to JOSS after this change. What do you think?

```suggestion
The datasets of the R programming language, such as those from the CRAN repository, are stored in the R specific formats Rda and Rds.
```

Reply:
```suggestion
The datasets of the R programming language, such as those from the CRAN repository, are often stored in the R specific formats Rda and Rds.
```

The sentence sounds like only these format could be used in R, so I'd suggest to add "often".

I agree that the paper is ready to be submitted to JOSS.